core/themes/basis/css/component/comment.css

```diff
@@ -163,3 +163,16 @@
 [dir="rtl"] .comment-footer {
   float: left;
 }
+
+a.comment-permalink  {
+  width: .7em;
+  height: .7em;
+  padding: .1em;
+  display: inline-block;
+  background-color: currentColor;
+  -webkit-mask-image: url("../../images/link-icon.svg");
```

Comment:
This needs a png fallback if we're going to use a SVG. Here's an example from messages where we also wanted to use an SVG. 
```
div.messages.info:before {
  background-image: url(../../../misc/message-32-info.png);
  background-color: #0074bd;
}
/* SVG overrides */
div.messages.info:before {
  background-image: linear-gradient(transparent, transparent), url(../../../misc/message-info.svg);
}
```

Reply:
> This needs a png fallback.

I'm not sure. Why? Internet Explorer? :laughing: (RIP)

@jenlampton this is different. It's not a background image, but a mask image. We use a mask as that way the icon color will follow the link color. Keep in mind: Basis has color module support, so the link color isn't necessarily blue, but can be something completely different.

See also: https://developer.mozilla.org/en-US/docs/Web/CSS/mask-image
and https://caniuse.com/?search=mask-image

Seems like SVG as mask have better support than PNG.

Reply:
Ah, I missed the mask image bit. If that's the case then maybe a fallback isn't necessary.